WikiDrips/WikiTableViewController.swift

```diff
@@ -52,13 +50,19 @@ class WikiTableViewController: UITableViewController, UISearchBarDelegate {
     }
     
     func search() {
-//      TODO: replace with async call to Wikipedia API
-        let someDateTime = Date(timeIntervalSinceNow: -(Double(wikiDocs.count) * 60))
-        let testData: [String: AnyObject] = ["title": "My new title" as AnyObject, "pubDate": someDateTime as AnyObject]
-        if let wikiDoc = WikiDoc(data: testData) {
-            wikiDocs.insert([wikiDoc], at: 0)
+        guard let searchText = searchText,
+            let wikiRequest = WikiRequest(searchText: searchText) else {
+            return
+        }
+
+        wikiRequest.fetchWikiDocs { newDocs in
+            DispatchQueue.main.async(){
+                if newDocs.count > 0 {
+                    self.wikiDocs = [newDocs]
```

Comment:
@nartiles, I saw your comment on capturing self here.  Just not sure how to handle that yet.  Any advice on this one?